README.md

```diff
@@ -11,21 +11,22 @@ ___
 
 This repositorty contains Swift implementation over [OpenAI](https://beta.openai.com/docs/api-reference/) public API.
 
-- [What is OpenAI](#what-is-openai)
-- [Installation](#installation)
-- [Usage](#usage)
+- [OpenAI](#openai)
```

Comment:
Could you clarify this change, please 🙏 
I think that we don't need all items to be grouped into a single one. What do you think?

Reply:
They are made by my VS code automatically. I'll reset these out.